README.md

```diff
@@ -1 +1,29 @@
-# ecommerce-team4
\ No newline at end of file
+# Ecommerce Project - Team4
+
+This website includes two parts one for customers and one for sellers , the site shows 
+all the avilable items the customer can search items by name and can filter them by
+price or category, he can add items to cart , delete them from cart and see total price 
+of items in cart.
+the seller can add items edit them or delete them, and can search in his products.
```

Comment:
the description is good, but how about we talk a little about the e-commerce site, ex: 

```
e-commerce is a responsive online shopping website, where the customer can see all the
available products and search for a specific item ... etc

```

**talk with other members to decide a name for the website!**

Reply:
ok